src/js/components/orgs/cards/orgActions.tsx

```diff
@@ -138,23 +140,76 @@ const OrgActions = ({
     const needsReview =
       task.has_unmerged_commits && task.pr_is_open && !task.review_valid;
     let isActive = false;
+    let popover = null;
+
     switch (type) {
       case ORG_TYPES.DEV:
         isActive = hasReviewRejected || !task.has_unmerged_commits;
+        popover = (
+          <TourPopover
+            align="top"
+            heading={i18n.t('Create a Dev Org')}
+            body={
+              <Trans i18nKey="tourTaskCreateDevOrg">
+                A Dev Org is a temporary Salesforce org where you can make
+                changes that you would like to contribute to the Project. To
+                create an Org, make sure you are connected to a Salesforce
+                account with Dev Hub enabled. Use the drop down menu to delete
+                the Dev Org when you no longer need it.
+              </Trans>
+            }
+          />
+        );
+
         break;
       case ORG_TYPES.QA:
         isActive = needsReview;
+        popover = (
+          <TourPopover
+            align="top"
+            heading={i18n.t('Create a Test Org')}
+            body={
+              <Trans i18nKey="tourTaskCreateTestOrg">
+                A Test Org is a temporary Salesforce org where you can view the
+                changes the Developer retrieved. To create an Org, make sure you
+                are connected to a Salesforce account with Dev Hub enabled. Read
+                the Developer’s Commit History to see what changes they made.
+                Use the drop down menu to delete the Test Org when you no longer
+                need it.
+              </Trans>
+            }
+          />
+        );
         break;
     }
     return (
       <>
-        {submitReviewBtn}
+        {submitReviewBtn ? (
+          <span className="slds-is-relative submit-btn">
+            {submitReviewBtn}
+            <TourPopover
+              align="right"
+              heading={i18n.t('Submit a review')}
+              body={
+                <Trans i18nKey="tourTaskSubmitReview">
+                  When you’re finished viewing and testing all the changes, come
+                  back to Metecho to leave your review. You will have the option
+                  to approve the work or request changes. Clearly describe any
+                  further changes you’d like the Developer to make.
+                </Trans>
+              }
+            />
+          </span>
+        ) : null}
         {!(preventNewTestOrg || disableCreation) && (
-          <Button
-            label={i18n.t('Create Org')}
-            variant={isActive ? 'brand' : 'neutral'}
-            onClick={doCreateOrg}
-          />
+          <span className="slds-is-relative task-create-btn">
```

Comment:
@dvdherron if we add `inline-container` here, then do we need the popover override for `task-create-btn`?

Reply:
adding `inline-container` worked. removed `task-create-btn`